src/components/filters/DateFilter/index.tsx

```diff
@@ -77,7 +77,7 @@ export const DateFilter: React.FC<Props> = (props) => {
         onChangeTo(event)
       }
 
-      const toDateMoment = moment(event.currentTarget.value).utc().add(24, 'hours')
+      const toDateMoment = moment(event.currentTarget.value).utc().endOf('day')
       const toTimestamp = toDateMoment.unix()
```

Comment:
Fix when we filter by date. Now we can filter in this way, for example
To: 16/10/2020 00:00:01
From: 16/10/2020 23:59:59